scipy/sparse/tests/test_base.py

```diff
@@ -760,6 +760,7 @@ def check(dtype, j):
             datsp = self.spmatrix(dat, dtype=dtype)
             assert_array_almost_equal(dat.sum(), datsp.sum())
             assert_equal(dat.sum().dtype, datsp.sum().dtype)
+            assert np.isscalar(datsp.sum(axis=None))
```

Comment:
This shouldn't be a bare `assert`, but ratherthe function `assert_`. Same goes for below.